src/Uno.UWP/Security.Credentials/PasswordVault.Android.cs

```diff
@@ -160,5 +164,36 @@ public override void Write(byte[] buffer, int offset, int count)
 					=> _adapter.Write(buffer, offset, count);
 			}
 		}
+
+		/// <summary>
+		/// Persister for devices bellow Android level 23.
+		/// RSA/ECB/PKCS1Padding only is supported and is not considered secure.
+		/// </summary>
+		private sealed class UnSecureKeyStorePersister : FilePersister
+		{
+			private const string _notSupported = @"RSA/ECB/PKCS1Padding with asymetric key is considered not secure and will not  be supported for device under API level 23";
+
+			private const string _lowLevelDeviceTransform = "RSA/ECB/PKCS1Padding";
+			private const string _provider = "AndroidKeyStore";
+			private const string _alias = "uno_passwordvault";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field '_alias'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826541)